src/elements.js

```diff
@@ -6,21 +6,20 @@ export let nodes = [];
 export let edges = [];
 export let nodesToBackend = []; // this version is nested and will be updated according to the ui changes
 
-export async function setNodes(nestedNodes) {
+export  function setNodes(nestedNodes) {
   nodesToBackend = JSON.parse(JSON.stringify(nestedNodes));
   nodes = calculatePosition(nestedNodes);
   console.log(nodes);
   //test:
-  let response = await fetch("/save", {
+  /*let response = await fetch("/save", {
     method: "POST",
     headers: {
       "Content-Type": "application/json",
     },
     body: JSON.stringify({
       yamlContent: JSON.stringify(nodesToBackend),
     }),
-  });
-  console.log(response);
+  });*/
```

Comment:
remove commented code - if it is for backup - we always have git to go back to find old versions of the code. there is no need to keep comments in the code for backup.